# Analytics Challange
 You are a business analyst working for ...:

 A business client has requested a large freight to be transported urgently from .... to .... When asked about the weight
 of the cargo they could not supply the exact weight, however they have specified that there are a total  of 36 boxes.

 From prior experience with this client you know that this type of cargo follows a distribution with mean=72 lbs and
 std.dev. = 3 lbs.

 Max cargo weight of a plane is 2630 lbs.

 Based on this information what is the probability that all of cargo can be safely loaded onto the plane and transported?

### Import necessary libraries

In [1]:
import numpy as np
from scipy import stats as st

### Assigning our data to variables

In [3]:
original_mean = 72
original_stdev = 3

number_of_samples = 36

max_cargo_weight = 2630

### Calculating sampling distribution mean, sampling distribution std dev

In [12]:
# original data mean = sampling distribution mean
# sampling distribution std dev = original data std dev / square root of sample numbers

sampling_distribution_mean = original_mean

sampling_distribution_stdev =  original_stdev / np.sqrt(number_of_samples)

critic_weight = round(max_cargo_weight / number_of_samples,2) # find out critic weight of one box so: critic weight box * number of boxes = max cargo weight of plane

print('Sampling distribution mean:', sampling_distribution_mean)
print('Sampling distribution std dev:', sampling_distribution_stdev)
print('Critic weight of a box:', critic_weight)

Sampling distribution mean: 72
Sampling distribution std dev: 0.5
Critic weight of a box: 73.06


### Find what percent probability the total weight is below the maximum cargo weight

In [14]:
# to find this probability we get help from z score
# z = (value - mean) / std dev

z_score_critic_weight = round((critic_weight - sampling_distribution_mean) / sampling_distribution_stdev, 2)

z_score_critic_weight

2.12

### That z score means critic weight of a box is 2.12 standart deviation away from mean. If we look at Standart Normal Probabilities Table we can see 2.12 z-score equals 0.9830 probability. So we can say that the cargo can be sent with a   98.3% probability.